### ISSUE TITLE
Question regarding the configuration for training the 'Edit' model (anycalib_op_e)

### ISSUE THREAD

**Issue by zoe265106-beep** [javrtg/AnyCalib#15]
First of all, congratulations on this excellent work! I am really impressed by the performance of AnyCalib.
I am currently trying to reproduce the training process for the Edit model (using anycalib_op_e), but I noticed a potential discrepancy between the source code and the provided configuration files.
I noticed that anycalib_net.py implements specific components designed for the Edit mode (to handle aspect ratio changes and principal point shifts):
The Decoder/Head:
In DECODERS, there is a light_dpt_tangent_edit_decoder key mapping to the ConvexTangentEditDecoder class. This decoder explicitly outputs pix_ar_map and radii alongside rays.
The Losses:
In the AnyCalib class (ray_loss method), there is logic to handle rays-laplace-ar-loss and rays-l1-r-loss specifically for supervising these auxiliary outputs.

However, the default configuration file (siclib/configs/model/anycalib.yaml) sets:
decoder.name: light_dpt_tangent_decoder (The base decoder)
loss.names: ['l1-z1'] (Only the geometric loss)
When running the training command for the Edit model:

python -m siclib.train anycalib_op_e --conf anycalib ...

It is not obvious how the configuration switches to the ConvexTangentEditDecoder and adds the auxiliary losses (laplace-ar and l1-r). Without these, the model wouldn't receive supervision for the pix_ar and radii branches.
My Question:
Is there a missing experiment config file (e.g., configs/experiments/anycalib_op_e.yaml) that overrides these defaults? Or does siclib handle this switch implicitly elsewhere?
I am currently manually overriding these parameters in my training script to ensure the Edit features are learned, but I wanted to confirm if this is the intended usage.
Thanks for your help!

**Comment by javrtg** [comment]
Thanks for the kind words and for the detailed question, and sorry for the very late response.

There is no missing experiment config. All released AnyCalib checkpoints (`anycalib_pinhole`, `anycalib_gen`, `anycalib_dist`, and `anycalib_edit`) were trained with the same architecture: DINOv2 + `ConvexTangentDecoder`. The difference between them is only in the training data used.

The `ConvexTangentEditDecoder` and the related auxiliary losses were implemented to try to improve the calibration accuracy from edited images. However, in our experiments we did not observe improvements from using them, so I would not recommend using that decoder.
